<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>630</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	color: rgb(0,0,127);
	background: transparent;
}

QTableWidget{
	background-color: rgba(255, 255, 255, 0);
}

QWidget#centralWidget{
	background-image: url(:/backgrounds/yande.re 64923.jpg);
}

QPushButton{
	background-color: rgba(255, 255, 255, 127);
	border: 1px solid white;
	border-radius: 5px;
	padding: 4 20 4 20px;
}

QPushButton:hover{
	background-color: rgba(255, 255, 255, 192);
}

QPushButton:pressed{
	background-color: rgba(255, 255, 255, 224);
}

QToolBox{
	background-color: rgba(255, 255, 255, 127);
}

QToolBox::Tab{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.5, y2:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}

QTableWidget{
	background-color: rgba(255, 255, 255, 127);
}

QHeaderView::section{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
border: 1px solid rgba(0, 0, 0, 32);
}

QScrollBar::add-page{
	background:transparent;
}

QScrollBar::sub-page{
	background:transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonLoad">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSaveAs">
        <property name="text">
         <string>SaveAs</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditCurrentFile">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color: rgba(255, 255, 255, 127);
	border: 1px solid rgba(255, 255, 255, 192);
	border-radius: 2px;
	padding: 1 1 1 1px;
}
QLineEdit:hover{
	background-color: rgba(255, 255, 255, 192);
	border: 1px solid rgba(255, 255, 255, 192);
	border-radius: 2px;
	padding: 1 1 1 1px;
}</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabSpacing">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageRigidBodies">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>882</width>
         <height>542</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="label">
        <string>RigdiBodies &gt;&gt;</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableRigidBodies">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::DotLine</enum>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>motion type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>shape</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="buttonAddBody">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRemoveBody">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageConstraints">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>882</width>
         <height>542</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Constraints &gt;&gt;</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableConstraints">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::DotLine</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>rigidbodies</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="buttonAddConstraint">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRemoveConstraint">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonLoad</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onButtonLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSaveAs</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onButtonSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableRigidBodies</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onModifyRigidBody(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableConstraints</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onModifyConstraint(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonLoad()</slot>
  <slot>onButtonSave()</slot>
  <slot>onModifyRigidBody(int,int)</slot>
  <slot>onModifyConstraint(int,int)</slot>
 </slots>
</ui>
